### ISSUE TITLE
tpatcher Optimization

### ISSUE THREAD

**Issue by armoha** [armoha/eudplib#25]
Tested in empty map, when `RemoveUnit(0, P1)` is changed to `RemoveUnit(eudv, P1)`, additional 37 triggers (objects) is used.
We can save triggers to less than 10 by using EUDX in tpatcher's filler functions.

```python
# eudplib\trigger\filler.py

def _dww(dstepd, v):
    act = c.Forward()
    c.SeqCompute(((ut.EPD(act + 16), c.SetTo, dstepd), (ut.EPD(act + 20), c.SetTo, v)))
    c.RawTrigger(actions=(act << c.SetMemory(0, c.SetTo, 0)))


def _filldw(dstepd, v1):
    c.SeqCompute(((dstepd, c.SetTo, v1),))


@c.EUDFunc
def _fillwbb(dstepd, v1, v2, v3):
    ret = c.EUDVariable()
    ret << 0

    for i in range(15, -1, -1):
        c.RawTrigger(
            conditions=v1.AtLeast(2 ** i),
            actions=[v1.SubtractNumber(2 ** i), ret.AddNumber(2 ** i)],
        )

    for i in range(7, -1, -1):
        c.RawTrigger(
            conditions=v2.AtLeast(2 ** i),
            actions=[v2.SubtractNumber(2 ** i), ret.AddNumber(2 ** (i + 16))],
        )

    for i in range(7, -1, -1):
        c.RawTrigger(
            conditions=v3.AtLeast(2 ** i),
            actions=[v3.SubtractNumber(2 ** i), ret.AddNumber(2 ** (i + 24))],
        )

    _dww(dstepd, ret)


@c.EUDFunc
def _fillbbbb(dstepd, v1, v2, v3, v4):
    ret = c.EUDVariable()
    ret << 0

    vlist = (v1, v2, v3, v4)
    for i, v in enumerate(vlist):
        lsf = 8 * i
        for i in range(7, -1, -1):
            c.RawTrigger(
                conditions=v.AtLeast(2 ** i),
                actions=[v.SubtractNumber(2 ** i), ret.AddNumber(2 ** (i + lsf))],
            )

    _dww(dstepd, ret)

```

**Comment by armoha** [comment]
Solved by 8134f6f76df3a2ee6b1794084bd5b6fc94ba64e4

**Comment by armoha** [comment]
Resolved in 70815d2c367702ef162a71998b3f2ff1048ba807
